tests/src/Kernel/Paragraphs/RichTextTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel\Paragraphs;
+
+use Drupal\filter\Entity\FilterFormat;
+use Drupal\paragraphs\Entity\Paragraph;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests the "Rich text" paragraphs.
+ */
+class RichTextTest extends ParagraphsTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+    // Create a text format and associate CKEditor.
```

Comment:
CKEditor is nowhere in this test, but even so, it is obvious what that code does so the comment is redundant